docs/pull_request_template.md

```diff
@@ -10,4 +10,6 @@
 - [ ] Documentation
     - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
     - [ ] JavaScript: basic explanation and screenshot of new features
+    - [ ] Text targeting end users (including first run)
+    - [ ] Text targeting developers (if relevant)
```

Comment:
```suggestion
    - [ ] Markdown docs for developers
```

"if relevant" not needed since authors can and should `[n/a]` bullets that don't apply.

Reply:
Applied